SPECS/python-pycryptodome/python-pycryptodome.spec

```diff
@@ -0,0 +1,419 @@
+#
+# spec file for package python-pycryptodome
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
+%define oldpython python
+Name:           python-pycryptodome
+Version:        3.14.1
+Release:        3
+Summary:        Cryptographic library for Python
+License:        BSD-2-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://www.pycryptodome.org
+Source:         https://github.com/Legrandin/pycryptodome/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  fdupes
+BuildRequires:  python-rpm-macros
```

Comment:
We don't ship this package anymore, since we removed python2. I would remove the line entirely, since `python3-devel` pulls in our python3 RPM macros.

Reply:
Right. I will remove this line.